interface/forms/vitals/growthchart/chart.php

```diff
@@ -79,9 +79,9 @@
 $first_datapoint = $datapoints[0];
 if (!empty($first_datapoint)) {
     $date = str_replace('-', '', substr((string) $first_datapoint['date'], 0, 10));
-    $height = (($isMetric) ? convertHeightToUs($first_datapoint['height']) : $first_datapoint['height']);
-    $weight = (($isMetric) ? convertWeightToUS($first_datapoint['weight']) : $first_datapoint['weight']);
-    $head_circ = (($isMetric) ? convertHeightToUs($first_datapoint['head_circ']) : $first_datapoint['head_circ']);
+    $height = $measurementUtils->isMetric() ? convertHeightToUs($first_datapoint['height']) : $first_datapoint['height'];
```

Comment:
What did the original `$isMetric` mean here? Did it mean the current value is metric and we need to convert it to imperial? Or did it mean the system is set to use metric, so we should convert all values to metric? It looks like this code is asking if something is set to metric, and then converting the value to "US" (Imperial). That's confusing.